### PR TITLE
Updated css selectors for vm actions tests

### PR DESCRIPTION
**Pull request by rhrazdil**
Changed css selectors of VM status object so that these are not dependent on page structure.

### CODE REVIEW

**Comment by vojtechszocs** [comment]
This PR depends on kubevirt/web-ui-components#198.

@rhrazdil Once the above mentioned PR gets merged, we'll need @mareklibra to bump & publish new version of `kubevirt-web-ui-components` package. Once it's published, please update the version in `kubevirt/package.json` to the latest :smiley:

**Comment by mareklibra** [comment]
@rhrazdil , web-ui-components version within web-ui is increased automatically as part of the web-ui-compoennets release process. You don't need to care explicitly.

File: kubevirt/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "dependencies": {
     "@patternfly/react-console": "1.9.1",
-    "kubevirt-web-ui-components": "0.1.16",
+    "kubevirt-web-ui-components": "0.1.17",
```
Comment (mareklibra) [inline kubevirt/package.json]: Please remove this explicit change here

Comment (rhrazdil) [reply kubevirt/package.json]: Okay, done.

**Comment by vojtechszocs** [comment]
> @rhrazdil , web-ui-components version within web-ui is increased automatically as part of the web-ui-compoennets release process. You don't need to care explicitly.

Thanks, that's good to know.

@rhrazdil I've improved kubevirt/web-ui-components#198 by adding CSS class to the `VmStatus` link itself (after seeing the impact of replacing `<Fragment>` with `<div>` on our React component test snapshots).

Please make the following change:

```diff
-export const detailViewVMmStatus = $('#details-column-1 .kubevirt-vm-status a');
-export const listViewVMmStatus = (name: string) => rowForName(name).$('.kubevirt-vm-status a');
+export const detailViewVMmStatus = $('#details-column-1 .kubevirt-vm-status__link');
+export const listViewVMmStatus = (name: string) => rowForName(name).$('.kubevirt-vm-status__link');
```

File: frontend/integration-tests/views/kubevirt/vm.actions.view.ts [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import { $, $$, browser, ExpectedConditions as until } from 'protractor';
 import { rowForName, confirmAction } from '../crud.view';
 
-export const detailViewVMmStatus = $('#details-column-1 > dl:nth-child(1) > dd:nth-child(2)');
-export const listViewVMmStatus = (name: string) => rowForName(name).$('div.co-m-row:first-child > div:first-child > div:nth-child(3)');
+export const detailViewVMmStatus = $('#details-column-1 .kubevirt-vm-status a');
```
Comment (vojtechszocs) [inline frontend/integration-tests/views/kubevirt/vm.actions.view.ts]: `#details-column-1 .kubevirt-vm-status__link`

File: frontend/integration-tests/views/kubevirt/vm.actions.view.ts [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
+export const listViewVMmStatus = (name: string) => rowForName(name).$('.kubevirt-vm-status a');
```
Comment (vojtechszocs) [inline frontend/integration-tests/views/kubevirt/vm.actions.view.ts]: `.kubevirt-vm-status__link`

**Comment by vojtechszocs** [comment]
kubevirt/web-ui-components#198 is now merged (thanks @rawagner) so this PR is no longer blocked.

File: frontend/integration-tests/views/kubevirt/vm.actions.view.ts (hunk 2 above)
Comment (rhrazdil) [reply frontend/integration-tests/views/kubevirt/vm.actions.view.ts]: done

File: frontend/integration-tests/views/kubevirt/vm.actions.view.ts (hunk 3 above)
Comment (rhrazdil) [reply frontend/integration-tests/views/kubevirt/vm.actions.view.ts]: done.

**Comment by rhrazdil** [comment]
@vojtechszocs updated the selectors + added the await keyword as we discussed.

**Comment by vojtechszocs** [comment]
Looks good, thanks @rhrazdil!

**Pull request merged**